site/src/docs/patterns/forms/form-building.mdx

```diff
@@ -328,528 +581,975 @@ Marking that indicates a form control for which **the user *must* enter a value*
-    <div class="guideline-dont-label" style="color:var(--color-error); margin-bottom:var(--spacing-s);"><IconError style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle'}} size="s"/><strong>Don't</strong></div>
-    <p><strong>Selections that demand the users consent, like approving terms and conditions <em>must</em> never be pre-selected as default.</strong></p>
+<div
+  class="guideline-dont"
+  style="background: var(--color-error-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;"
+>
+  <div class="guideline-dont-label" style="color:var(--color-error); margin-bottom:var(--spacing-s);">
+    <IconError style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle' }} size="s" />
+    <strong>Don't</strong>
+  </div>
+  <p>
+    <strong>
+      Selections that demand the users consent, like approving terms and conditions <em>must</em> never be pre-selected
+      as default.
+    </strong>
+  </p>
 </div>
 
 ---
 
 ## Providing instructions
+
 To help the user enter the **right information in the right format**, guidance can be provided on different levels:
 
 1. **Form main and sub-heading** summarise the purpose of the form
 2. **Form-wide instructions** to help the user to better understand the functionality of the form
 3. **Contextual instructions** for helping the user to fill specific form controls
 
-
 ### 1. Form heading and sub-heading
-The form heading **summarises the purpose of the form** and indicates the user where the form begins. 
 
-- The *main heading should* clearly and concisely **describe the purpose of the form**. 
-    - Full page forms *should* always have a main heading. 
-    - From smaller inline forms embedded to page content, the heading can be omitted.
-- If the purpose of the form cannot be sufficiently summarised by the main heading, it can **elaborated with a *sub-heading***.
-
-<Image src="/images/patterns/form/building/form-guiding-headings@2x.png" alt="Form headings" style="max-width:432px;" viewable /> 
-
-
-<div class="guideline-do" style="background: var(--color-success-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;">
-    <div class="guideline-do-label" style="color:var(--color-success); margin-bottom:var(--spacing-s);"><IconCheck style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle'}} size="s"/><strong>Do</strong></div>
-    <ul style="margin-bottom:0;">
-        <li><strong>If the form has a main heading, it <em>must</em> be the first element</strong>
-            <ul style="margin-top:0;">
-                <li>The main heading <em>should</em> preferably be h1 element, even if visually styled as a smaller level heading.</li>
-            </ul>
+The form heading **summarises the purpose of the form** and indicates the user where the form begins.
+
+- The _main heading should_ clearly and concisely **describe the purpose of the form**.
```

Comment:
Prettier has changed also these but it does not break them

Reply:
So this is ok? Looks good on site